features/openhab-core/src/main/feature/feature.xml

```diff
@@ -30,6 +32,7 @@
         <feature>openhab-runtime-certificate</feature>
         <feature>openhab-transport-mdns</feature>
         <feature>openhab-transport-http</feature>
+        <feature>esh-io-transport-http-auth-basic</feature>
```

Comment:
esh-io-transport-http-auth-basic is renamed as per [this](https://github.com/eclipse/smarthome/commit/da1dda7cd04fd5d49d78834c6cb20799713f4fb9) commit.